### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by disciple-zarrin**
add Dockerfile and .env

### CODE REVIEW

**Comment by disciple-zarrin** [comment]
update dockerfile and .env

**Review by AnonymousWP — COMMENTED** [review]
What are you doing? First, the `.env` file shouldn't be commited as it exposes private information, just like how I now know the password of your database, reNgine URL and where you live. See: https://github.com/yogeshojha/rengine/pull/890.

Furthermore: why are you making changes to the Dockerfile related to the name, author and description? This project is called reNgine and belongs to Yogesh Ojha, not you or your company. Also, substantiate why you made those changes in the Dockerfile.

Long story short: completely useless PR.
